src/templates/_stripView.hbs

```diff
@@ -1,7 +1,20 @@
+<script src="assets/scripts/client/jquery.min.js"></script>
+<script src="assets/scripts/client/jquery-ui.min.js"></script>
+  <script>
```

Comment:
This is another big no-no.  Absolutely no inline javascript.

Any js should live within the existing codebase and *never* in our templates.  The only exception here is our google analytics snippet.